sdk/core/core-http/src/policies/deserializationPolicy.ts

```diff
@@ -87,14 +96,15 @@ export class DeserializationPolicy extends BaseRequestPolicy {
       (deserializationContentTypes && deserializationContentTypes.json) || defaultJsonContentTypes;
     this.xmlContentTypes =
       (deserializationContentTypes && deserializationContentTypes.xml) || defaultXmlContentTypes;
+    this.xmlCharKey = parsingOptions?.xmlCharKey || XML_CHARKEY;
```

Comment:
might as well be consistently fancy, right? ;) 

```suggestion
    this.xmlCharKey = parsingOptions?.xmlCharKey ?? XML_CHARKEY;
```